source/components/simpleCardList/simpleCard.ts

```diff
@@ -79,6 +80,8 @@ export class SimpleCardController implements ISimpleCardBindings {
 				this.close();
 			}
 		});
+
+		this.cardType = this.cardType != null ? this.cardType : null;
```

Comment:
FYI - you actually don't need this. Angular will automatically set the cardType via a binding. This was just if you wanted to have a default value